src/ml_board/frontend/Dashboard/src/components/graphs/Graph.tsx

```diff
@@ -98,6 +169,12 @@ export default function Graph({ chart_id, exp_id, exp_data }: { chart_id: string
                         options={options}
                     />
                 </Box>
+                {
+                    location.pathname.split("/")[1] !== RoutesMapping["ExperimentPage"].url ?
```

Comment:
**location** vs **redux** vs **localstorage** ?

Reply:
removed location from everywhere as said above. Local storage is used for auto-populating the config popup as we discussed in the meeting.